### ISSUE TITLE
Intense flickering on models when fighting

### ISSUE THREAD

**Issue by Oscarg73** [tlaik/neptastic#6]
Hi! First, many thanks for this amazing mod.

I have a problem: when fighting, models (allies and enemies) not participating on the turn should simply go white / transparent.
![20210729101708_1](https://user-images.githubusercontent.com/25150787/127485125-9b32f676-f98c-4ef8-b342-b24cf779cc2e.jpg)
![20210729101713_1](https://user-images.githubusercontent.com/25150787/127485137-db793597-bd40-4ab5-8812-6bfb76e76593.jpg)

But with the mod, they start flickering like crazy. Maybe it's a problem with the 144hz refresh rate or the 2K resolution of my screen, I don't know. Tried to record the screen and I only got 60FPS so the models just disappear (it's more noticeable on Ram's turn).

https://user-images.githubusercontent.com/25150787/127485711-c98d087b-331d-4a9e-8f5d-11017fdaf436.mp4

I also add my sepcs in case it would be of any help:

Screen: 2560x1440 / 144hz
OS: Windows 10 Pro 21H1
CPU: AMD Ryzen 9 5900X 12-Core Processor 3.70 GHz
RAM: 32GB DDR4 3200MHz
GPU: NVIDIA GTX 1070ti 8GB GDDR5

**Comment by tlaik** [comment]
This should be easy enough to test by switching off the FPS unlock in the config. Try it and see if the enemies will look like in the base game again.

**Comment by Oscarg73** [comment]
> This should be easy enough to test by switching off the FPS unlock in the config. Try it and see if the enemies will look like in the base game again.

Hi! Yes, capping the FPS to 60 solves the problem.
![image](https://user-images.githubusercontent.com/25150787/127488813-b8333484-f33e-4002-a3ed-e4b12a996232.png)
![20210729140139_1](https://user-images.githubusercontent.com/25150787/127488933-a4c3134a-868e-42b6-ada1-5683df6a0e9f.jpg)

But hey! I also tried R;B3 without the mod (because it's the only one with unlimited FPS) and I got that intense flickering too. So it seems that's a problem of the game, not your mod!

https://user-images.githubusercontent.com/25150787/127489486-2ca708f8-2d9d-497b-9828-a600f234d350.mp4
